tests/annotations/tests.py

```diff
@@ -405,6 +405,26 @@ def test_order_by_aggregate(self):
             lambda a: (a['age'], a['age_count'])
         )
 
+    def test_child_with_raw_sql_annotation_on_parent(self):
+        DepartmentStore.objects.create(
+            name='Angus & Robinson',
+            original_opening=datetime.date(2014, 3, 8),
+            friday_night_closing=datetime.time(21, 00, 00),
+            chain='Westfield',
+        )
+        tests = (
+            ('name', ()),
+            ("case when name='foo' then 'oof' else name end", ())
```

Comment:
We should check if this statement works instead of returning `name` field, e.g.

```python
def test_raw_sql_with_inherited_field(self):
    DepartmentStore.objects.create(
        name='Angus & Robinson',
        original_opening=datetime.date(2014, 3, 8),
        friday_night_closing=datetime.time(21),
        chain='Westfield',
    )
    tests = (
        ('name', 'Angus & Robinson'),
        ("case when name='Angus & Robinson' then chain else name end", 'Westfield'),
    )
    for sql, expected_name in tests:
        with self.subTest(sql=sql):
            self.assertSequenceEqual(
                DepartmentStore.objects.annotate(
                    title=RawSQL(sql, ()),
                ).values_list('title', flat=True),
                [expected_name],
            )
```